From 11aa3ea6c7d4448c4cc1d07523079f3f4d3dc800 Mon Sep 17 00:00:00 2001
From: Srikanth Krishnakar <Srikanth_Krishnakar@mentor.com>
Date: Fri, 28 Nov 2014 14:02:56 +0530
Subject: [PATCH] systemd-udevd: propagate mounts/umounts services to system

JIRA: SB-4128

"MountFlags=slave" restricts the access of the mounted/unmounted
directories.Other way to propagate the mount services to entire
system is to share.

MountFlags=shared will automount the media devices.

Signed-off-by: Srikanth Krishnakar <Srikanth_Krishnakar@mentor.com>
---
 units/systemd-udevd.service.in |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: git/units/systemd-udevd.service.in
===================================================================
--- git.orig/units/systemd-udevd.service.in	2015-09-14 11:18:31.364305816 -0700
+++ git/units/systemd-udevd.service.in	2015-09-14 11:18:40.296351938 -0700
@@ -21,6 +21,6 @@
 Restart=always
 RestartSec=0
 ExecStart=@rootlibexecdir@/systemd-udevd
-MountFlags=slave
+MountFlags=shared
 KillMode=mixed
 WatchdogSec=1min
